system/config/Config.cfc

```diff
@@ -980,6 +980,14 @@ component {
 		};
 
 		settings.rest.authProviders.token = { feature = "restTokenAuth" };
+
+		settings.rest.apis[ "/presideUtils" ]   = {
+			  authProvider        = "presideApi"
+			, description         = "Utility REST API for external systems to interact with Preside"
+			, dataApiQueueEnabled = false
```

Comment:
Data API is not part of core Preside and cannot be used here. I think you just need to remove `, dataApiQueueEnabled = false`

Reply:
I think we also ought to have a feature flag on this that would be disabled by default. Either that, or we move this whole feature out into a different extension (e.g. our test data ext).